app/src/main/java/org/simple/clinic/search/results/PatientSearchResultsScreen.kt

```diff
@@ -76,16 +77,15 @@ class PatientSearchResultsScreen(context: Context, attrs: AttributeSet) : Relati
     setupAlertResults(screenDestroys)
   }
 
-  @SuppressLint("CheckResult")
```

Comment:
Can we add this back? It now shows a warning that we are not handling the result of the subscription.